go/interpreter/lfvm/interpreter.go

```diff
@@ -20,18 +20,18 @@ import (
 	"github.com/holiman/uint256"
 )
 
-type Status byte
+type status byte
```

Comment:
since field status is really good fit for all it's uses, perhaps this type could be named `statusType` or `statusT` ? wdyt ?

I am aware this is totally legal in go, but I think having variables/fields named the same as their types, is just propense to confusion.

Reply:
I would argue that it is not good style (and we have not done this before) to encode in an identifier the nature of the concept that is defined. For instance, we do not go for the convention that all interfaces should start with an `I` or that constants have a `k` prefix. I think not having to say that some identifier is a type by not adding a `T` or `Type` into its name falls in the same category.